config.yaml

```diff
@@ -10,8 +10,9 @@ claimKeyBanDuration: 1
 # so 14 days ago is the oldest data.
 maxDiagnosisKeyRetentionDays: 15
 
-# A generated keypair can upload up to 28 keys (14 on day 1, plus 14 subsequent days)
-initialRemainingKeys: 28
+# A generated keypair can upload up to 43 keys (15 on day 1, plus 2 for 14 subsequent days
+# if they upload once per day)
```

Comment:
This should be 13 * today’s partial I think? Unless it’s 13 past + 2 for today.

Regardless, a lot of assumptions .

Reply:
14 day previous + 1 fo day 0, and then 28. The last upload only is the partial day.